Dockerfile

```diff
@@ -12,18 +12,18 @@ RUN apt-get update && apt-get install -y $RUNTIME_PACKAGES $BUILD_PACKAGES \
 	&& curl -sL https://deb.nodesource.com/setup_5.x | bash -
 
 RUN apt-get install -y nodejs
-RUN npm install --global gulp-cli
```

Comment:
Doesn't look like yarn has actually been added as a dependency here. Guess we need to add the same packages to sources list and apt-install as per the travis file?